tests/tests.js

```diff
@@ -42,10 +41,6 @@ var isAudioSupported = isWindows
         ? !window.SAUCELABS_ENV
         : true;
 
-var isKitKat =
-    cordova.platformId === 'android' &&
-    /Android\s4\.4/.test(window.navigator.userAgent);
-
```

Comment:
This might be considered breaking?

Might want to considered merging this PR after the versions is major bumped...

Reply:
Fortunately, it's just the test code